### ISSUE TITLE
I would like to be able to deploy DbKVS backed by Oracle.

### ISSUE THREAD

**Issue by jboreiko** [palantir/atlasdb#608]

**Comment by jboreiko** [comment]
Linked to #384

**Comment by cooper87** [comment]
I wish to turn it into reality too. Today I try to start Atlasdb with "atlasdb_standalone_postgres.yml", but failed. The console of eclipse print : 
```
src/dist/var/conf/atlasdb_standalone_postgres.yml has an error:
  * Failed to parse configuration at: keyValueService; Could not resolve type id 'postgresdb' into a subtype of [simple type, class com.palantir.atlasdb.spi.KeyValueServiceConfig]: known type ids = [KeyValueServiceConfig, cassandra, memory, partitioned, rocksdb]
 at [Source: N/A; line: -1, column: -1] (through reference chain: com.palantir.atlasdb.config.Json["keyValueService"])
```
How it would be wonderful if Atlasdb support postgres, because it will be easier to learn it for newcomers .

**Comment by cooper87** [comment]
Maybe I start Atlasdb in the wrong way, would you like to introduce the steps of starting it with postgres?

**Comment by gsheasby** [comment]
Pull request: #985.

**Comment by gsheasby** [comment]
Will be accomplished by #1076

**Comment by gsheasby** [comment]
Fixed by #1076
